pkg/operator/ceph/cluster/osd/health.go

```diff
@@ -122,8 +122,13 @@ func (m *Monitor) handleOSDMarkedOut(outOSDid int) error {
 			return fmt.Errorf("failed to get osd deployment of osd id %d: %+v", outOSDid, err)
 		}
 		if len(dp.Items) != 0 {
-			if err := k8sutil.DeleteDeployment(m.context.Clientset, dp.Items[0].Namespace, dp.Items[0].Name); err != nil {
-				return fmt.Errorf("failed to delete osd deployment %s: %+v", dp.Items[0].Name, err)
+			podCreationTimestamp := dp.Items[0].GetCreationTimestamp()
+			podDeletionTimeStamp := podCreationTimestamp.Add(60 * time.Minute)
```

Comment:
Can we use a constant for this instead?

Reply:
I didn't get how a constant will be helpful here

Reply:
Using a constant will make reading easier and that value more meaningful as well as modifying it since it's at the top.

Reply:
Agreed, it is easier to see a constant at the top of the file.